devicetypes/smartthings/zwave-basic-window-shade.src/zwave-basic-window-shade.groovy

```diff
@@ -58,11 +58,26 @@ metadata {
 				defaultValue: false,
 				displayDuringSetup: false
 			)
+            		
+			input(title: "Aeotec Nano Shutter settings",
+			    description: "Sets the Open/Close time.",
+			    displayDuringSetup: false,
+			    type: "paragraph",
+			    element: "paragraph")
+
+			input("calibrationTime", "number",
+			    title: "Set the open/close time for motor",
+			    defaultValue: false,
+			    displayDuringSetup: false,
+			    range: "5..255",
+			    default: 10
```

Comment:
While you're here, could you just refactor how all of these settings are presented? I think we could consolidate them quite a bit.

I'm thinking it should just be:
```
			input("reverseDirection", "bool",
				title: "Reverse working direction",
                                 description: "In case wiring is wrong, this setting can be changed to fix setup without any manual maintenance.",
				defaultValue: false,
				displayDuringSetup: false
			)            	

			input("calibrationTime", "number",
                             title: "Open/Close timing"
			    description: "Set the motor's open/close time",
			    defaultValue: false,
			    displayDuringSetup: false,
			    range: "5..255",
			    default: 10
			)
```
(but don't just copy/paste that because the spacing will be off)

I'd also add a small comment next to both of these just noting that they're for the Aeotec Nano Shutters in case we ever add another device to this DTH.

Reply:
@Greens - not a problem here will have this edited soon.

Reply:
@greens - Edited and setting this to resolved.